### ISSUE TITLE
can´t compile in ASC 2.0

### ISSUE THREAD

**Issue by Seb-AS** [fljot/GestouchExamples#6]
hi Pavel

i´m using flash builder 4.7 and the last gestouch  swc
I have a problem with these line but in flash cs6 works fine.
Gestouch.inputAdapter ||= new NativeInputAdapter(stage);
TypeError: Error #1034: Error forced conversion: you can't convert  in org.gestouch.core.IInputAdapter.
thanks 
Sebas

**Comment by fljot** [comment]
Yup, ASC 2.0 doesn't seem smart enough with "mario mushroom operator" (http://www.tillschneidereit.de/2011/10/22/mario-mushroom-now-with-80-less-awesome/)
Small discussion here http://forum.starling-framework.org/topic/gestouch-%E2%80%94-multitouch-gestures-for-starling-and-classic-displaylist/page/2

**Comment by xpepermint** [comment]
``` ruby
Gestouch.inputAdapter = new NativeInputAdapter(stage); 
```

will work.
